thinkhazard/static/less/common.less

```diff
@@ -17,6 +17,7 @@
 @input-border-focus: #58595b;
 @input-border-radius: 5px;
 
+@border-radius-small: @input-border-radius;
```

Comment:
I wouldn't reuse the `@input-border-radius` here. This variable is used in Bootstrap. This means that any input will have a radius.
We want to do the same thing for the small buttons. So actually, what we want to override here is: `@btn-border-radius-small`.
So I would write the following instead:
```less
@btn-border-radius-small: 5px;
```
I know, I'm picky.